server/localization/sk_SK.json

```diff
@@ -0,0 +1,13 @@
+{
+    "help_message": "%devider%Dostupne prikazy:\n  %.help% - tento help\n  %.look% - napise co vidi\n  %.go% - presunie ta do lokacie\n  %.pick% <item name> - zdvihne <item name>\n  %.drop% <item name> - zahodi <item name>\n  %.combine% <item name>, <item name>, ... - skombinuje zadane polozky\n  %.recepies% - vypise recepty // TODO\n  %.say% - povie do lokacie // DISABLED\n  %.yell% - povie vsetkym hracom // DISABLED\n  %.stats% - ukaze tvoje statistiky\n  %.challenge% - vyzvi na suboj hraca\n    %.attack% <num> - zautoci s hodnotou <num>\n    %.buff% - buffnes svoje sance na dalsi utok\n  %.r% - zopakuje posledny prikaz",
+    "server_welcome_message": "Ahoj, %player%",
+    "server__player_interaction_name": "Vitaj, napis si meno:",
+    "server__player_interaction_death": "Dead man tells no tales",
+    "server__player_interaction_inv_location": "Invalid location, did you spell it right?",
+    "server__player_interaction_inv_command": "Invalid command! use %.help%",
+    "server__player_interaction_disabled_command": "This command is disabled by an administrator!",
+
+    "in_game_translations": {
```

Comment:
I believe it would be better if localization json stays flat, without nested values.
If nesting is wanted, keys can contain dots to separate namespaces or variants. For example:

```json
{
  "server.welcome_message": "Ahoj, %player%",
  "server.player_interaction.death": "Zomrel si",
  "item.lego_brick.one": "lego kocka",
  "item.lego_brick.few": "lego kocky",
  "item.lego_brick.many": "lego kociek"
}
```